modules/core/src/GeneratingConcept.js

```diff
@@ -35,6 +35,10 @@ export default class GeneratingConcept extends OverridingConcept {
     /** @type {Record<string, any>} */
     const recordsServer = {}
 
+    if (this.generateMultipleFiles) {
```

Comment:
Copying concept has getter called removesDestinationDirectory. That can be set to true/false if we want to remove old files before generating. Now we can see that Copying concept isnt the only concept in need of this functionality. So it would be better to lift it from copying concept to some concept higher in the hierarchy so it can be reused.